lib/td/client/api.rb

```diff
@@ -251,7 +251,7 @@ def do_get(url, params=nil, &block)
     unless ENV['TD_CLIENT_DEBUG'].nil?
       puts "DEBUG: REST GET call:"
       puts "DEBUG:   header: " + header.to_s
-      puts "DEBUG:   path:   " + path.to_s
+      puts "DEBUG:   path:   " + (@base_path + url).to_s
```

Comment:
td-client-ruby(0.8.77) shows `target` that set L251.
Which do would be appropriate to be displayed?